atomix/values.yaml

```diff
@@ -55,3 +59,14 @@ resources:
   requests:
     cpu: 0.5
     memory: 512Mi
+
```

Comment:
might be worth not running with enabled security context by default but only tuning it on in production. @kuujo thoughts ?

Reply:
no because default should be a secure setup and if a user wants to make it insecure he should do it explicitly otherwise we provide a non-secure definition by default

Reply:
no because default should be a secure setup and if a user wants to make it insecure he should do it explicitly otherwise we provide a non-secure definition by default